server/middleware/workstack.js

```diff
@@ -66,14 +66,26 @@ function workstackApiResponseMiddleware(req, res) {
     res.json(res.locals.workstack);
 }
 
+const updateCaseDataMoveTeamRequest = async (req, res, [endpoint, body, headers]) => {
+    const logger = getLogger(req.requestId);
+    try {
+        await workflowService.put(endpoint, body, headers);
+        return;
```

Comment:
These `return undefined` doesn't serve any purpose as it's at the end of the function.

Reply:
True will remove.

Reply:
This has been rectified.